data/pizza.json

```diff
@@ -989,8 +989,8 @@
 				"hidden_ingredients": [
 					"tomatsås", "ost"
 				],
-				"price": 110,
-				"family_price": 240
+				"price": 115,
+				"family_price": 245
 			},
 			"pesto": {
```

Comment:
[The fårost strikes again](https://sannegarden.com/meny/johanneberg/#:~:text=Champinjoner%2C%20F%C3%A5rost%2C%20Ruccolasallad%2C%20Soltorkade%20Tomater%2C%20Mozzarellaost%2C%20Pesto%2C%20Kyckling)